ibm/service/power/resource_ibm_pi_instance.go

```diff
@@ -972,42 +978,42 @@ func isPIInstanceSoftwareLicensesRefreshFunc(client *st.IBMPIInstanceClient, id
 		// Check that each software license we modified has been updated
 		if softwareLicenses.IbmiCSS != nil {
 			if *softwareLicenses.IbmiCSS != *pvm.SoftwareLicenses.IbmiCSS {
-				return pvm, "notdone", nil
+				return pvm, StatusNotDone, nil
```

Comment:
I had put done/notdone, but please feel free to change it if there a status that sounds more appropriate for waiting of the ibmi licenses to update.